source/parser.hera

```diff
@@ -80,15 +85,17 @@ ForbiddenImplicitCalls
   "/ "
   AtAt # experimentalDecorators
 
-IndentedApplicationAllowed
-  "" ->
-    if (module.suppressIndentedApplication) return $skip
-    return
-
 ArgumentsWithTrailingMemberExpressions
-  # Since this is recursive Arguments must consume input to avoid infinite recursion
+  Arguments TrailingMemberExpressions
+
+TrailingMemberExpressions
   # NOTE: Assert "." to not match "?" or "!" as a member expression on the following line
-  Arguments ( Samedent &"." MemberExpressionRest )*
+  MemberExpressionRest* ( ( Samedent / IndentedFurther ) &"." MemberExpressionRest )* ->
```

Comment:
Personally I think `?.` and maybe even `!.` could make sense as a trailing member expression.

Reply:
I think the preferred place for `!` would be as a suffix after the identifier but `?.` makes sense to me.